strr-examiner-web/app/pages/registration/[registrationId].vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+const { t } = useI18n()
+const route = useRoute()
+const { manageAction } = useExaminerActions()
+const { updateRouteAndButtons } = useExaminerRoute()
+const { updateRegistrationStatus, getRegistrationById } = useExaminerStore()
+
+useHead({
+  title: t('page.dashboardList.title')
+})
+
+definePageMeta({
+  layout: 'examine',
+  middleware: ['auth']
+})
+
+const initialMount = ref(true)
+
+const { data: registration, status, error, refresh } = await useLazyAsyncData<
+  HousRegistrationResponse | undefined, ApplicationError
+>(
+  'registration-details-view',
+  async () => {
+    // slug will be there, otherwise the route will not be rendered and redirected to dashboard
+    const slug = Number(route.params.registrationId)
+    return await getRegistrationById(slug)
+  }
+)
+
+const handleRegistrationAction = (
+  id: number,
+  action: 'CANCEL' | 'SUSPEND',
+  buttonPosition: 'left' | 'right',
+  buttonIndex: number
+) => {
+  const status = action === 'CANCEL' ? RegistrationStatus.CANCELLED : RegistrationStatus.SUSPENDED
+  return manageAction(
+    { id },
+    action,
+    updateRegistrationStatus,
+    buttonPosition,
+    buttonIndex,
+    refresh,
+    [status]
+  )
+}
+
+watch(
+  [registration, error],
+  ([_, __]) => {
```

Comment:
`[_, __]` is not needed here (and above I saw it too), it is not used. Could be `() => {}`

Reply:
Done